packages/react-testing/README.md

```diff
@@ -0,0 +1,267 @@
+# `@shopify/react-testing`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-testing.svg)](https://badge.fury.io/js/%40shopify%2Freact-testing.svg)
+
+A library for testing React components according to our conventions.
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-testing
+```
+
+## Usage
+
+This library is broadly similar to [Enzyme](https://github.com/airbnb/enzyme), but with a focus on type safety and testing based on a component’s external API. In order to keep the API small and easy-to-use, it will generally track to only the latest minor release of React.
+
+### <a name="mount"></a> `mount(element: React.ReactElement<any>)`
+
+Mounts a component to the DOM and returns a [`Root`](#root) instance. Note that in order for this to work, you must have a simulated browser environment, such as the `jsdom` environment that Jest uses.
+
+Unlike Enzyme, there are no options to `render` or `shallow` render a component. These patterns do not work well under many circumstances, so there is no API in this library to use them.
```

Comment:
Don't bother talking about or comparing to Enzyme at all IMO.

I could see a separate FAQ part in this readme that is like "How does this library compare to Enzyme?" or "Why use this library over Enzyme?" where the differences are listed. But in the main part of this Readme, just focus quickly on the things this library does and consider that some new devs might not even know what Enzyme is.